include/solvers/navier_stokes_base.h

```diff
@@ -74,16 +74,22 @@
 #include <deal.II/fe/fe_q.h>
 #include <deal.II/fe/fe_system.h>
 #include <deal.II/fe/fe_values.h>
+#include <deal.II/fe/mapping_fe.h>
 #include <deal.II/fe/mapping_q.h>
 
 // Numerics
+#include <deal.II/base/quadrature_lib.h>
+
+#include <deal.II/fe/fe_simplex_p.h>
```

Comment:
Move this line above with the other fe (or apply automatic indentation to fix it)